tests/queries/0_stateless/02500_remove_redundant_distinct.reference

```diff
@@ -46,18 +46,17 @@ ORDER BY number
 Expression (Projection)
   Distinct
     Sorting (Sorting for ORDER BY)
-      Distinct (Preliminary DISTINCT)
```

Comment:
The reason of the change is that `Preliminary DISTINCT` was pushed down over `UNION`.
I am not sure whether we should remove it in this case, but this was done by `remove_redundant_distinct` optimization. (why?)